tutorials/market/Market_Intelligence_Part4.ipynb

```diff
@@ -0,0 +1,1840 @@
+{
```

Comment:
<p>Very nice results! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/197/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>